.github/workflows/CI-python.yml

```diff
@@ -119,4 +112,4 @@ jobs:
           name: codecov-umbrella
           path_to_write_report: ./coverage/codecov_report.txt
           verbose: true
-        if: ${{ matrix.packageDirectory != 'raiwidgets'}}
+        if: ${{ always()}}
```

Comment:
This seems like it'll always evaluate to true so you can probably remove the line